CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md

```diff
@@ -4,21 +4,55 @@ Customization guide
 The Gini Capture SDK components can be customized either through the `GiniConfiguration`, the `Localizable.string` file or through the assets. Here you can find a complete guide with the reference to every customizable item.
 
 - [Generic components](#generic-components)
+- [Onboarding screens](#onboarding-screens)
+- [No Authorized screen](#no-authorized-screen)
 - [Camera screen](#camera-screen)
+- [Gallery album screen](#gallery-album-screen)
 - [Review screen](#review-screen)
-- [Multipage Review screen](#multipage-review-screen)
 - [Analysis screen](#analysis-screen)
-- [Supported formats screen](#supported-formats-screen)
-- [Open with tutorial screen](#open-with-tutorial-screen)
-- [Capturing tips screen](#capturing-tips-screen)
-- [Gallery album screen](#gallery-album-screen)
-- [Onboarding screens](#onboarding-screens)
-- [Help screen](#help-screen)
+- [Help screens](#help-screens)
+- [No result screen](#no-result-screen)
+- [Error screens](#error-screens)
+
+## Colors
+
+We are providing a global color palette `GiniColors.xcassets` which you are free to override. The custom colors will be then applied on all screens.
+You can find the names of the colors [https://github.com/gini/gini-mobile-ios/tree/new-ui/CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/GiniColors.xcassets](https://github.com/gini/gini-mobile-ios/tree/new-ui/CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/GiniColors.xcassets).
+
+ You can view our color palette here:
+
+<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="600" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FH4WFJ4xaw4YNU4VaJYWiQq%2FiOS-Gini-Capture-SDK-2.0.0-UI-Customisation%3Fnode-id%3D14%253A355%26t%3DwpenBBM8QsagJzOg-1" allowfullscreen></iframe>
```

Comment:
[Suggestion] I would extract only the link from here, and put it in a nicer format, right now if somebody opens it in a viewer, it looks like this: 

<img width="862" alt="image" src="https://user-images.githubusercontent.com/19169669/202994052-b7cd8ea7-5685-499e-8660-2d5d9482a63a.png">


It would be nice to have it like this maybe?

You can view our color palette [here](https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FH4WFJ4xaw4YNU4VaJYWiQq%2FiOS-Gini-Capture-SDK-2.0.0-UI-Customisation%3Fnode-id%3D14%253A355%26t%3DwpenBBM8QsagJzOg-1)

Reply:
I tried it out and saw that after rendering these markdown pages into html the embedded iframe will work and it will look like this:
![Screenshot 2022-11-21 at 11 08 15](https://user-images.githubusercontent.com/13149498/203023547-8a399c8a-3e1c-4b6e-aff3-f0741b9bc687.png)

You can try out the rendered documentation by going to the `Checks` tab, then `Build docs for Capture SDK` and in the `Artifacts` section download the `GiniCaptureSDKDocumentation`.

Reply:
This snippet is needed to see a preview of the palette:
<img width="1009" alt="Screenshot 2022-11-21 at 11 13 01" src="https://user-images.githubusercontent.com/3911816/203024606-e33ab723-8cea-4156-938f-4ec59209c351.png">